### PR TITLE
Chore: Changed ButtonBase min-width and Number Input border radius

### PR DESCRIPTION
**Pull request by RakeshUP**
Task: [APP-1349](https://aragonassociation.atlassian.net/browse/APP-1349), [APP-1352](https://aragonassociation.atlassian.net/browse/APP-1352)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [x] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

[APP-1349]: https://aragonassociation.atlassian.net/browse/APP-1349?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-1352]: https://aragonassociation.atlassian.net/browse/APP-1352?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/ui-components/src/components/button/buttonBase.tsx [hunk 1]
```diff
@@ -44,12 +44,18 @@ ButtonBase.displayName = 'ButtonBase';
...
+const sizeClassNames = {
   small: 'h-4 space-x-1 rounded-lg',
   medium: 'h-5 space-x-1.5 rounded-larger',
   large: 'h-6 space-x-1.5 rounded-xl',
 };
 
+const sizeStyles = {
+  small: {minWidth: '32px'},
+  medium: {minWidth: '40px'},
+  large: {minWidth: '48px'},
```
Comment (Fabricevladimir) [inline packages/ui-components/src/components/button/buttonBase.tsx:56]: Just curious, are we unable to do this with Tailwind?

**Review by Fabricevladimir — APPROVED** [review]

File: packages/ui-components/src/components/button/buttonBase.tsx (hunk 1 above)
Comment (RakeshUP) [reply packages/ui-components/src/components/button/buttonBase.tsx:56]: Tailwind doesn't have classes for arbitrary values of min-width. Only if we were using tailwind v3...

Comment (RakeshUP) [reply packages/ui-components/src/components/button/buttonBase.tsx:56]: Tailwind doesn't have classes for arbitrary values of `min-width`. Only if we were using tailwind v3...

Comment (Fabricevladimir) [reply packages/ui-components/src/components/button/buttonBase.tsx:56]: Ah, makes sense. Thanks

**Review by WissenIstNacht — APPROVED** [review]

**Pull request merged**